epaint/src/textures.rs

```diff
@@ -49,8 +49,9 @@ impl TextureManager {
             if delta.is_whole() {
                 meta.size = delta.image.size();
                 meta.bytes_per_pixel = delta.image.bytes_per_pixel();
+                // since we update the whole image, we can discard all old enqueued deltas
```

Comment:
seems like an odd comment to add considering the new code keeps the old deltas :)

Reply:
Good catch! I altered the code a bit to comply with the comment. My reasoning here would be: removing a couple of elements from the vector of texture deltas is likely less computational intensive than altering a texture a bunch of times only the to replace it completely.